### PR TITLE
WQP-1576 (update WQP projects to work with new artifactory)

### PR DESCRIPTION
**Pull request by orosz-usgs**
Change urlRules priority pipeline.yml from 302 to 320 to avoid conflicting
with other project that already has a priority of 302.

Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: pipeline.yml [hunk 1]
```diff
@@ -38,4 +38,4 @@ urlRules:
       - /Result/*
       - /Station/*
       - /Activity/*
-    priority: 302
+    priority: 320
```
Comment (ayan-usgs) [inline pipeline.yml:41]: What's the reason for changing this?

Comment (orosz-usgs) [reply pipeline.yml:41]: When deploying to prod-external, the deployed failed with:
Priority '302' is currently in use (Service: AmazonElasticLoadBalancingV2; Status Code: 400; Error Code: PriorityInUse; Request ID: 47e35162-a641-44fd-a2e8-d3e80e4920d9)

Comment (ayan-usgs) [reply pipeline.yml:41]: Change it back to `302`. It's failing because the stack drifted due to manual changes.

Comment (orosz-usgs) [reply pipeline.yml:41]: We can just close this pull request then. The Jenkins deploy job is failing:
File "/var/lib/jenkins/jobs/IOW/jobs/WQP/jobs/SDLC/jobs/WQP-WQX-Services-deploy/workspace/aws/utils.py", line 238, in execute_change_set
    raise ValueError('{} status received'.format(stack_status))
ValueError: UPDATE_ROLLBACK_IN_PROGRESS status received

**Pull request closed without merging**